<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GlassPropertyDlg</class>
 <widget class="QDialog" name="GlassPropertyDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>183</width>
     <height>349</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Dispersion</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="12" column="1">
     <widget class="QLineEdit" name="lineEdit_Dispersion_C9"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_Dispersion_C1"/>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>C8</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>C0</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="lineEdit_Dispersion_C8"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_Dispersion_C3"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>C6</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>C2</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_Dispersion_C2"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>C1</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_Dispersion_C5"/>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="lineEdit_Dispersion_C7"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>C4</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>C5</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="lineEdit_Dispersion_C4"/>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lineEdit_Dispersion_C6"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>C9</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>C3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_Dispersion_C0"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>C7</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>Formula</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Dispersion_Formula"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>182</width>
     <height>225</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Thermal Data</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_E1"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_E0"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_D1"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_D2"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_D0"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_18">
      <property name="text">
       <string>D0</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>D2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_22">
      <property name="text">
       <string>E1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>D1</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>E0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_23">
      <property name="text">
       <string>Ltk</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_Ltk"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>221</width>
     <height>194</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Refractive Index</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="2" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>ne</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_index_C"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_index_e"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_index_g"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>nC</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>nd</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>ng</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_index_d"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>nF</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_index_F"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>221</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="title">
    <string>Fundamentals</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_status"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_24">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>status</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_Supplyer"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_26">
      <property name="text">
       <string>Nd</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_GlassName"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Supplyer</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Glassname</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_Nd"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_Vd"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_27">
      <property name="text">
       <string>Vd</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
